docs/opup.rst

```diff
@@ -0,0 +1,85 @@
+.. _opup:
+
+Budget Increase Utility
```

Comment:
@algoidurovic Optionally, I propose mentioning _OpUp_ in the page title.  Since it seems we'll use the language _OpUp_, I think it'd be helpful for readers to see the term in the table of contents + at top of page.

```suggestion
OpUp:  Budget Increase Utility
```